fdbserver/ClusterRecovery.actor.cpp

```diff
@@ -191,6 +198,7 @@ ACTOR Future<Void> newCommitProxies(Reference<ClusterRecoveryData> self, Recruit
 		req.recoveryCount = self->cstate.myDBState.recoveryCount + 1;
 		req.recoveryTransactionVersion = self->recoveryTransactionVersion;
 		req.firstProxy = i == 0;
+		req.encryptMode = getEncryptionAtRest(self->configuration);
 		TraceEvent("CommitProxyReplies", self->dbgid)
 		    .detail("WorkerID", recr.commitProxies[i].id())
 		    .detail("RecoveryTxnVersion", self->recoveryTransactionVersion)
```

Comment:
nit: please log the encyrptionMode details as well?

Reply:
Actually i can just add it should be fine